src/Microsoft.Health.Fhir.Core/Features/Conformance/CapabilityStatementBuilder.cs

```diff
@@ -248,6 +248,10 @@ public ICapabilityStatementBuilder PopulateDefaultResourceInteractions()
                     if (!string.Equals(resource, KnownResourceTypes.AuditEvent, StringComparison.Ordinal))
                     {
                         component.ConditionalUpdate = true;
+
+                        component.ConditionalDelete.Add(ConditionalDeleteStatus.NotSupported);
+                        component.ConditionalDelete.Add(ConditionalDeleteStatus.Single);
+                        component.ConditionalDelete.Add(ConditionalDeleteStatus.Multiple);
```

Comment:
[https://www.hl7.org/fhir/capabilitystatement-definitions.html#CapabilityStatement.rest.resource.conditionalDelete](https://www.hl7.org/fhir/capabilitystatement-definitions.html#CapabilityStatement.rest.resource.conditionalDelete)

Cardinality 0..1

We should put only one value here.

Also I would update comment in line 247 to reflect what delete is not supported for AuditEvent as well.